README.md

```diff
@@ -196,15 +210,13 @@ const lab = exports.lab = Lab.script();
 
 const suite = lab.suite;
 const test = lab.test;
-const before = lab.before;
```

Comment:
to comply with `no-unused-vars`, I deleted `before` and `after` here and used them in the BDD section to split the difference